docs/developer-guide/syntax-reference.rst

```diff
@@ -306,6 +306,22 @@ defines a struct with three fields, with the supplied name and type.
 Access the fields of a struct by using the ``->`` operator. For example, ``SOME_STRUCT->ID``
 retrieves the value of the struct's ``ID`` field. For more information, see :ref:`operators`.
 
+Decimal
+-------
+
+``DECIMAL(Precision, Scale)``
+
+KSQL supports fields that are numeric data types with fixed precision and scale:
+
+- **Precision** is the maximum total number of decimal digits to be stored, including values to
+  the left and right of the decimal point. The precision must be greater than 1 - there is no
+  default precision.
+- **Scale** is the number of decimal digits to the right of the decimal points. This number must
+  be greater than 0 and less than the value for Precision.
```

Comment:
```suggestion
  be greater than 0 and less than, or equal to, the value for Precision.
```

Right?  e.g. `.123` has scale and precision of `3`.

Reply:
yup :)